pkg/webhooks/sdnmigration/sdnmigration.go

```diff
@@ -24,6 +24,7 @@ const (
 var (
 	log                         = logf.Log.WithName(WebhookName)
 	privilegedServiceAccountsRe = regexp.MustCompile(utils.PrivilegedServiceAccountGroups)
+	priviledgeUserAccount       = "admin-kubeconfig-signer"
```

Comment:
Since we are specifically referring to this as the `hive` user, let's name it as such. Also type in `priviledge`, versus `priviledged`. Something like `privilegedHiveUserAccount`.

Also, this should be a constant as it's a `string`, not a `var`

Reply:
Change done